content/docs/tasks/security/vault-ca/index.md

```diff
@@ -154,3 +155,123 @@ between workloads using the certificates the Vault CA issued.
 After completing this tutorial, you may delete the testing cluster created
 at the beginning of this tutorial.
 
+## Appendix
+
+Vault servers are hosted and managed by their owners. When signing a CSR, Istio Citadel Agent
```

Comment:
Done. I chose to start here because it clarifies that Istio does not manage or configure Vault server. The owner of a Vault server is responsible of managing, configuring, and securing the server.